be/src/olap/delta_writer.cpp

```diff
@@ -38,6 +38,9 @@ DeltaWriter::~DeltaWriter() {
     if (!_delta_written_success) {
         _garbage_collection();
     }
+    for (SegmentGroup* segment_group : _segment_group_vec) {
+        segment_group->release();
+    }
```

Comment:
`_garbage_collection` and this code will release a segment group twice

Reply:
OK

Reply:
why `_new_segment_group_vec` doesn't need release?